src/wp-includes/class-wp-scripts.php

```diff
@@ -863,6 +863,18 @@ private function get_dependents( $handle ) {
 		return $dependents;
 	}
 
+	/**
+	 * Determines if a scripts defined loading strategy is valid.
```

Comment:
```suggestion
	 * Determines if a script loading strategy is valid.
```

Reply:
@joemcgill @kt-12 I addressed this as part of a recent commit. I also changed the param description to be more reflective of your suggestion @joemcgill